spec/ts-smoke/electron/main.ts

```diff
@@ -1072,17 +1072,17 @@ shell.writeShortcutLink('/home/user/Desktop/shortcut.lnk', 'update', shell.readS
 
   // Query all cookies.
   session.defaultSession.cookies.get({})
-    .then((cookies) => {
+    .then((cookies: Electron.Cookie[]) => {
```

Comment:
these types (all the changes in this file) should be automatically inferred..?

Reply:
Yeah if these aren't being inferred anymore we have a big problem